src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/EventsHandler.php

```diff
@@ -382,33 +202,155 @@ private function processOrderOnlyEvents($events)
         foreach ($events as $id => $event) {
             $details = $event['details'] ?? [];
             $eventType = $event['event'];
-            switch ("$module::$eventType") {
+
+            $fullEventType = "$module::$eventType";
+
+            $this->handleOrderEvents($fullEventType, $taskData, $details);
+
+            switch ($fullEventType) {
```

Comment:
maybe change the stock as well to function? than consistent with the rest

Reply:
Moved to methdo `handleProductStockEvents`.